lib/new_relic/harvest/collector/metric_data.ex

```diff
@@ -253,5 +253,20 @@ defmodule NewRelic.Harvest.Collector.MetricData do
       }
     ]
 
+  def transform(:queue_time, duration_s: nil), do: []
+
+  def transform(:queue_time, duration_s: duration_s) do
```

Comment:
@binaryseed I'm struggling with this metric in particular. When it was in the transaction metrics, I could easily harvest it in tests to make sure it was dispatched. I'm not sure how to test that it's dispatched now, though.

Reply:
Cancel.